### ISSUE TITLE
SetTouchEnabled disables clicking and touch events in static fragment

### ISSUE THREAD

**Issue by tunjid** [umano/AndroidSlidingUpPanel#580]
In version 3.0.0, my SlidingUpPanelLayout had an embedded static fragment via XML. This fragment hosted a RecyclerView, and to support scrolling of the RecyclerView, I would setTouchEnabled(false) so the panel could only be shown by tapping a button, and hidden by pressing the back button. The RecyclerView scrolled with no issue since touch was disabled.

Now in 3.2.0 with the scrollable view support, it's now broken as I cannot set the umanoScrollableView on the static fragment. If I call setTouchEnabled(false), I can't interact with the nested fragment. No touch, scroll or anything.

My XML looks like this:

```xml
<com.sothree.slidinguppanel.SlidingUpPanelLayout
    android:id="@+id/sliding_layout"
    xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    xmlns:fab="http://schemas.android.com/apk/res-auto"
    xmlns:tools="http://schemas.android.com/tools"
    android:layout_width="match_parent"
    android:layout_height="match_parent"
    android:gravity="bottom"
    app:umanoOverlay="true"
    app:umanoPanelHeight="0dp"
    app:umanoShadowHeight="0dp">

    <android.support.design.widget.CoordinatorLayout
        android:id="@+id/coordinator_layout"
        android:layout_width="match_parent"
        android:layout_height="match_parent">

        <include
            android:id="@+id/action_bar"
            layout="@layout/toolbar_base"
            android:layout_width="match_parent"
            android:layout_height="@dimen/abc_action_bar_default_height_material"
            android:layout_gravity="top" />

        <RelativeLayout
            android:id="@+id/container"
            android:layout_width="match_parent"
            android:layout_height="match_parent" />

        <FloatingActionButton
            android:id="@+id/fab"
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            android:layout_gravity="bottom|end"
            android:layout_margin="16dp"
            android:elevation="@dimen/quarter_margin"
            android:src="@drawable/ic_add_white_24dp"
            android:visibility="gone"
            fab:pressedTranslationZ="12dp" />
    </android.support.design.widget.CoordinatorLayout>

    <fragment
        android:id="@+id/search_fragment"
        android:name="com.myApp.StaticSearchFragment"
        android:layout_width="match_parent"
        android:layout_height="match_parent"
        android:layout_gravity="start"
        android:tag="SEARCH_TAG"
        tools:layout="@layout/fragment_search" />
</com.sothree.slidinguppanel.SlidingUpPanelLayout>
```

**Comment by akadimi** [comment]
Same thing here for programmatically added fragment

**Comment by tokudu** [comment]
Fixed in https://github.com/umano/AndroidSlidingUpPanel/commit/966fee4f049c43c1fded6371d1501d761f043395. Will be released in 3.3
